index.html

```diff
@@ -1,175 +1,105 @@
-        <p>Para um amor existir no futuro, primeiro é preciso imaginá-lo</p>
-O AMFA é uma instalação artística virtual  que compila cartas de pessoas para o futuro amor, ou o grande amor, e também uma intervenção urbana, que espalha lambes dessas cartas pela cidade. O projeto nasce na cidade de Salvador - Bahia, e invoca seus interlocutores (e possíveis futuros colaboradores) a compartilharem perspectivas poéticas da idéia, (ou existência), de futuros/possíveis/grandes amores.
-<p></p>
-      </footer>
-    <script>
-      const form = document.getElementById("form-carta")
-
-      form.addEventListener("submit", function(event) {
-        event.preventDefault();
-        const formData = new FormData(this);
-        const entries = formData.entries();
-        const data = Object.fromEntries(entries);
-
-        console.log({ data })
-
-        fetch("/", {
-          method: "POST",
-          body: JSON.stringify(data),
-          headers: {
-            "Content-Type": "application/json"
-          }
-        })
-          .then(res => { 
-            document.getElementById("resultado").innerHTML = "Carta enviada!"
-          })
-          .catch(/**/);
-      });
-    </script>
+      </button>
+      <div id="about-modal" class="modal">
+         <!-- Modal content -->
+        <p id="about-content">
+          <span class="close" id="close-about">&times;</span>
+            O AMFA é uma instalação artística virtual que compila cartas de
+            pessoas para o futuro amor, ou o grande amor. Se extende como uma
+            intervenção urbana, que espalha lambes dessas cartas pela cidade. O
+            projeto nasce na cidade de Salvador - Bahia, e invoca seus
+            interlocutores (e possíveis futuros colaboradores) a compartilharem
+            perspectivas poéticas da idéia, (ou existência), de
+            futuros/possíveis/grandes amores.
+        </p>
+      </div>
+      <br />
+      <br />
+      <button class="bttn-2" id="write-bttn">
+        <div class="overlap-group">
+          <div class="bttn-2-text">envie sua carta</div>
+        </div>
+      </button>
+      <div id="form-modal" class="modal">
+        <span class="close" id="close-form">&times;</span>
+        <div class="form-container flex column" id="form-content">
+          <form id="form-letter" class="form flex column">
+            <label for="txt-area" class="statement">
+              Escreva aqui a sua carta para um futuro amor</label
+            >
+            <br />
+            <textarea
+              type="text"
+              id="txt-area"
+              class="box"
+              name="letter"
+              maxlength="6000"
+              rows="10"
+              cols="100"
+              required
+              spellcheck="false"
+             ></textarea>
+            <br />
+            <button class="bttn-3" type="submit">Enviar para o futuro</button>
+          </form>
+          <p id="result"></p>
+        </div>
+      </div>
```

Comment:
If you want to dig deeper in HTML and JS, I would recommend making the modals (form and about modals) a web component so you can abstract the open/close behaviour within it.

See:
- https://developer.mozilla.org/en-US/docs/Web/API/Web_components
- https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements

🤗

Reply:
Let's do it! I'm excited to work with web components! :sparkles: